src/Uno.UWP/Helpers/ProtocolActivation.wasm.cs

```diff
@@ -24,29 +24,31 @@ public static void RegisterCustomScheme(string scheme, Uri domain, string prompt
 			// rules as per https://developer.mozilla.org/en-US/docs/Web/API/Navigator/registerProtocolHandler
 
 			// The custom scheme's name begins with web+
-			if (scheme.IndexOf("web+", System.StringComparison.InvariantCulture) != 0)
+			if (!scheme.StartsWith("web+", StringComparison.Ordinal))
```

Comment:
- `IndexOf` is slower than `StartsWith`, though I can see that for correct arguments this will not matter.
- Also changed string comparison to what I think more appropriate.